devices.js

```diff
@@ -10914,6 +10914,24 @@ const devices = [
             await reporting.onOff(endpoint);
         },
     },
+    {
+        zigbeeModel: ['RGBgenie ZB-5028'],
+        model: 'ZB-5028',
+        vendor: 'RGB Genie',
+        description: 'RGB remote with 4 endpoints and 3 scene recalls',
+        fromZigbee: [fz.battery, fz.command_on, fz.command_off, fz.command_step, fz.command_move, fz.command_stop, fz.command_recall,
+            fz.command_move_hue, fz.command_move_to_color, fz.command_move_to_color_temp],
+        exposes: [e.battery(), e.action(['on', 'off', 'brightness_step_up', 'brightness_step_down', 'brightness_move_up',
+            'brightness_move_down', 'brightness_stop', 'recall_*'])],
```

Comment:
Based on the description containing: 3 scene recalls , I think `recall_*` can be replaced by `recall_1`, `recall_2` and `recall_3`.

Reply:
Will do, tnx.